src/05_lists.py

```diff
@@ -8,22 +8,31 @@
 
 # Change x so that it is [1, 2, 3, 4]
 # YOUR CODE HERE 
+x.append(4)
 print(x)
 
 # Using y, change x so that it is [1, 2, 3, 4, 8, 9, 10]
 # YOUR CODE HERE 
+x.extend(y)
 print(x)
 
 # Change x so that it is [1, 2, 3, 4, 9, 10]
-# YOUR CODE HERE 
+# YOUR CODE HERE
+x.pop(4)
```

Comment:
python also has a remove where it will remove that item from the list. It would be helpful if you didn't know what index that item was at. 
`x.remove(8)`